src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/health/QueueHealth.kt

```diff
@@ -1,13 +1,9 @@
 package uk.gov.justice.digital.hmpps.keyworker.services.health
 
 import com.amazonaws.services.sqs.AmazonSQS
-import com.amazonaws.services.sqs.model.GetQueueAttributesRequest
-import com.amazonaws.services.sqs.model.GetQueueAttributesResult
-import com.amazonaws.services.sqs.model.GetQueueUrlResult
-import com.amazonaws.services.sqs.model.QueueAttributeName
+import com.amazonaws.services.sqs.model.*
```

Comment:
I believe we're avoiding wildcard imports now (I know this is the IDE doing things).